roles/traffic_control/templates/unit.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Apply network limits using tc (traffic control)
+After=network.target
+
+[Service]
+Type=oneshot
```

Comment:
`Type=oneshot` will consider the service as started once the script ends succesfully. This is exactly what we want. Our firewall role works the same way.

Re, why using systemd if it's just a script: We benefit that it will automatically start on bootup and time it right, in this case after the networking has come up, that's why there's `After=network.target`. Additionally, it will log everything to journald, which makes it easy to debug. Start/Stop also becomes easy, as you just do `systemctl stop/start/status traffic-control`.